test/test_python_dispatch.py

```diff
@@ -10,6 +10,52 @@
 
 import logging
 
+class TestPythonRegistration(TestCase):
+    def test_override_cpu_sum(self) -> None:
+        # Example 1
+        run = [False]
+        def my_sum(*args, **kwargs):
+            run[0] = True
+            return args[0]
+
+        torch.ops.aten.sum.default.impl("CPU", my_sum)
+
+        x = torch.tensor([1, 2])
+        self.assertEqual(torch.sum(x), x)
+        self.assertTrue(run[0])
+
+        # Example 2
+        def my_neg(*args, **kwargs):
+            return args[0]._neg_view()
+
+        # Now we are secretly making the operator a view op so autograd needs to know how
+        # to handle it
+        torch.ops.aten.neg.default.impl("AutogradCPU", my_neg)
```

Comment:
How do I know I should use "AutogradCPU" here?
Alban: you need more info about dispatcher.

Reply:
Ed: Is there a way to distinguish if I am adding a new kernel for a dispatch key which didn't have any impl or overriding an existing kernel (wanna use it for adding support for meta tensors)